src/SprykerEco/Zed/Payone/Business/Payment/MethodMapper/KlarnaMapperInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\Payment\MethodMapper;
+
+use Generated\Shared\Transfer\PayoneKlarnaStartSessionRequestTransfer;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\ContainerInterface;
+use SprykerEco\Zed\Payone\Business\Payment\PaymentMethodMapperInterface;
+
+interface KlarnaMapperInterface extends PaymentMethodMapperInterface
+{
+    /**
+     * {@inheritDoc}
```

Comment:
* The specification and `@api` tag are not needed here.

Reply:
Removed